buildroot-external/patches/occu/0154-WebUI-DynamicResizing.patch

```diff
@@ -0,0 +1,282 @@
++  border-collapse: collapse;  
++}
++
++#maintable tbody > tr > td {
++  padding: 0;
++}
++
++#maintable #content {
++  height: 100%;
++  background-repeat: no-repeat;
++  background-position: 480px center;
++}
++
+ /* Navigation-Leiste */
+ #navbar 
+ {
++  height: 72px;
+   margin: 0px;
+   background-color: $_(background); /* #183473 */
+   width:100%;
+@@ -144,6 +165,10 @@
+ }
+  
+ /* Haupt-Fenster */
++.Layer0 {
++  height: 100%;
++}
++
+ #_content
+ {
+   height: 70%;
+@@ -177,6 +202,8 @@
+     align-items: flex-start;
+     /* Prevent chrome from outlining this by focus */
+     outline: none;
++    /* Stretch to max available space */
++    height: calc(100% - 168px); /* FIX: header + menubar + footer: 72px + 41px + 55px = 168px */
+ }
+ 
+ #prgcontent
+@@ -196,7 +223,7 @@
+ 
+ /* Kopfzeile */
+ #header {
+-  height: 15%;
++  height: 72px;
+   background-color: $_(background);  /* #183473 */
+   /* background-color: #dcdcdc; */
+   color: $_(textColorB); /* black */
+@@ -206,7 +233,7 @@
+ 
+ /* Fusszeile */
+ #footer {
+-  height:15%;
++  height: 50px;
+   background-color: $_(background); /* #183473 */
+   /* background-color: #dcdcdc; */
+   color: $_(textColorB); /* black */
+@@ -254,12 +281,14 @@
+ /* Linker Bereich der Hauptansicht (falls in zwei Spalten getrennt) */
+ #contentLeft 
+ {
++  height: 100%;
+   vertical-align: top;
+ }
+ 
+ /* Rechter Bereich der Hauptansicht (falls in zwei Spalten getrennt) */
+ #contentRight 
+ { 
++  height: 100%;
+   border-left: solid 5px $_(navBarSeparator); /* #ffffff */ /* Trennlinie zwischen Navigation und Hauptbereich */
+   /* padding-top: 10px; */
+   vertical-align: top;
+@@ -2551,7 +2580,7 @@
+ }
+ #menubar {
+   height: 34px;
+-  width: 100%;
++  /* width: 100%; */
+   border-bottom:solid 5px $_(navBarSeparator); /* #ffffff */
+   background-color: $_(background); /* #183473 */
+   z-index: 189;
+@@ -6274,6 +6303,7 @@
+ }
+ 
+ .CLASS03901 {
++  height: 100%;
+ 	width:100%;
+ }
+ 
+@@ -6335,6 +6365,7 @@
+ }
+ 
+ .CLASS04001 {
++  height: 100%;
+ 	width:100%;
+ }
+ 
+--- occu/WebUI/www/webui/webui.js.orig
++++ occu/WebUI/www/webui/webui.js
+@@ -6942,6 +6942,8 @@
+    */
+   onResize: function()
+   {
++    /**
++     * not required any more, since we moved to css styles
+     var height       = WebUI.getHeight();
+     var width        = WebUI.getWidth();
+     var bodyOverflow = "hidden";
+@@ -6955,6 +6957,7 @@
+     if ($("menubar")) { Element.setStyle("menubar", {"height": this.MENUBAR_HEIGHT + "px", "width": width + "px"}); }
+     if ($("content")) { Element.setStyle("content", {"height": contentHeight       + "px", "width": width + "px"}); }
+     if ($("footer"))  { Element.setStyle("footer" , {"height": this.FOOTER_HEIGHT  + "px", "width": width + "px"}); }
++    */
+ 
+     if (this.currentPage) { this.currentPage.resize(); }
+ 
+@@ -20235,6 +20238,8 @@
+    **/
+   resizeUser: function()
+   {
++    /**
++     * not required any more, since we moved to css styles
```

Comment:
No stray comments, if not required please remove it altogether. And best would be to get rid of the whole `resizeUser()` function if possible.

Reply:
done, see 9a27326